github/common.js

```diff
@@ -113,6 +113,21 @@ function augment(a) {
   }
 }
 
+function augmentTooltip(a) {
+  const authorElement = document.querySelector("a.author.pull-header-username");
+  const who = a.getAttribute("aria-label");
+
+  if (!(authorElement && who)) {
+    return;
+  }
+
+  const author = authorElement.textContent.trim();
+  if (who.includes(author)) {
+    a.setAttribute("redacted-label", who.replace(author, "[redacted]"));
```

Comment:
Why is this needed?  Is there an example tooltip that contains additional text, other than just the author name?

Reply:
yes this contains `author/REPO:branch` and turns into `[redacted]/REPO:branch`

Reply:
I guess you're asking why not just set it to `"[redacted]"`? Robustness, maybe to much here. But better safe than sorry :D

Reply:
Ok, if it's unused, let's drop in the spirit of YAGNI, and keep both js and css simpler.

Reply:
Just to get this right:

Adding the `redacted-label` attribute is needed. I can change this to:
```js
a.setAttribute("redacted-label", "[redacted]");
```

if you think the `replace` is to much here. I can also replace the:
```js
if (who.includes(author)) {
```
with
```js
if (who === author) {
```

I'd argue both of them are clean right now. But I guess that's a matter of preference. You decide.

Reply:
I don't care much between `===` or `includes`, but the attribute doesn't seem to be needed, so we can just hardcode the `[redacted]` string in css.

Reply:
I see what you meant. However GitHub made all of this obsolete last week: https://blog.github.com/2018-03-22-user-hovercards/

Thanks anyhow. I'll refactor #33

Reply:
Ouch, and thanks.